lib/eventFields.js

```diff
@@ -6,7 +6,7 @@ import {
   GraphQLSchema,
 } from 'graphql';
 
-import { EventFetch, AllEventsFetch } from './eventFetch.js';
+import { fetchOne, fetchAll } from './eventOrNewsFetch.js';
```

Comment:
Can we not name the file to just `fetch.js`? It should be oblivious to specific document types.